doc/frameworks/tensorflow/using_tf.rst

```diff
@@ -197,6 +197,26 @@ The following args are not permitted when using Script Mode:
 Where the S3 url is a path to your training data within Amazon S3.
 The constructor keyword arguments define how SageMaker runs your training script.
 
+Create an Estimator using Docker containers
```

Comment:
I think this sounds a bit misleading - the estimator isn't being created by way of somehow running a Docker container - this is just describing how to specify a custom Docker image with an estimator.

Also, what's the motivation for putting this in the TF doc as opposed to a more general section? The information included in this change is more broad than TF.